From: Lionel Elie Mamane <lmamane@debian.org>
Date: Mon, 4 Jul 2016 11:13:17 +0200
Subject: Keep upstream build system from doing distro work

 1) Don't recreate spec file: unused on Debian,
    pollutes diff with ever-changing timestamp
    that changes result of rebuilds.
 2) Don't automatically generate debian/changelog,
    nor touch debian/ in any way.
---
 configure.in | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

Index: sqliteodbc/configure.in
===================================================================
--- sqliteodbc.orig/configure.in
+++ sqliteodbc/configure.in
@@ -748,8 +748,4 @@ AC_SUBST(SQLITE4_A10N_O)
 #########
 # Generate the output files.
 #
-SPEC_CHANGELOG_TS=`LANG=C date -u "+%a %b %d %Y"`
-DEB_CHANGELOG_TS=`LANG=C date "+%a, %d %b %Y %X %z"`
-AC_SUBST(SPEC_CHANGELOG_TS)
-AC_SUBST(DEB_CHANGELOG_TS)
-AC_OUTPUT(Makefile sqliteodbc.spec debian/changelog)
+AC_OUTPUT(Makefile)
